planner/core/preprocess.go

```diff
@@ -417,13 +422,24 @@ func isTableAliasDuplicate(node ast.ResultSetNode, tableAliases map[string]inter
 	return nil
 }
 
-func isPrimary(ops []*ast.ColumnOption) int {
+func checkColumnOptions(ops []*ast.ColumnOption) (int, error) {
+	isPrimary := 0
+	isGenerated := 0
+
```

Comment:
`isPrimary, isGenerated := 0, 0` is better

Reply:
@imtbkcat PTAL. Thanks.